.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
When you create a new repo on github, it can generate a biggish .gitignore for you, and I recommend using that, it will likely reduce annoyances later on.

Reply:
This one? https://github.com/github/gitignore/blob/master/Node.gitignore

Reply:
yup